packages/schematics/angular/angular-app/index.ts

```diff
@@ -103,6 +109,8 @@ export default function (options: any): Rule {
     addBootstrapToNgModule(options.directory),
     mergeWith(
       apply(url('./other-files'), [
+        options.minimal || options.inlineTempalte ?
+          filter(path => !path.endsWith('.html')) : noop(),
```

Comment:
No, this is only because we replace the template with the default app one from the `other-files` dir.